### ISSUE TITLE
Exception in update_project_resources method

### ISSUE THREAD

**Issue by mikhels** [marklap/rundeckrun#22]
Hi, faced with the following problem when tried to update resources:

```
...
new_node = {"name": "some_node", "hostname": "some_node.some_domain", "username": "some_user"}
rd.update_project_resources(project, nodes=[new_node])

Traceback (most recent call last):
  File "rundeck_test.py", line 20, in <module>
    rd.update_project_resources(project, nodes=[new_node])
  File "/usr/local/lib/python2.7/dist-packages/rundeckrun-0.1.12-py2.7.egg/rundeck/transforms.py", line 258, in wrapper
    results = func(self, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/rundeckrun-0.1.12-py2.7.egg/rundeck/client.py", line 964, in update_project_resources
    if isinstance(node, dict) and set(d.keys()).issuperset(required_keys):
NameError: global name 'd' is not defined
```

Seems that 'node' variable should be used instead of 'd'. Can you please check and fix it? Thank you in advance!

**Comment by marklap** [comment]
I think you're right. I pushed a fix in branch `issue22`. Can you test and let me know if it works?

**Comment by mikhels** [comment]
Works great now. Thank you!

**Comment by marklap** [comment]
Excellent. I'll cut a release soon.

**Comment by marklap** [comment]
Resolved with 18914f620dbb640fd17792da28b173cd320f1d7a
